nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/CountText.java

```diff
@@ -141,7 +141,15 @@
             .allowableValues(getStandardCharsetNames())
             .defaultValue(StandardCharsets.UTF_8.displayName())
             .build();
-
+    public static final PropertyDescriptor ADJUST_IMMEDIATELY = new PropertyDescriptor.Builder()
+            .name("ajust-immediately")
+            .displayName("Call Immediate Adjustment")
+            .description("If enabled, it will call the processSession.adjustCounter method to expose the counters immediately.")
```

Comment:
Most readers are just NiFi users and maybe don't know what `processSession.adjustCounter` method actually does. How about the following instead? Copied from ProcessSession.adjustCounter method API doc:

> If true, the counter will be updated immediately, without regard to whether the ProcessSession is commit or rolled back; otherwise, the counter will be incremented only if and when the ProcessSession is committed.

Then it'd be clearer that enabling this will affect when a session is rolled back.

Reply:
Okay, I'm going to change the description for that one.

Reply:
done.